pkg/api/apikey.go

```diff
@@ -19,10 +19,10 @@ import (
 //
 // Will return auth keys.
 //
-// Deprecated: true.
+// Deprecated. Please refer to [updated API](#/service_accounts/listTokensD) instead
+// see https://grafana.com/docs/grafana/latest/administration/service-accounts/migrate-api-keys/.
 //
-// Deprecated. Please use GET /api/serviceaccounts and GET /api/serviceaccounts/{id}/tokens instead
-// see https://grafana.com/docs/grafana/next/administration/service-accounts/migrate-api-keys/.
+// Deprecated: true
```

Comment:
The dot and `Deprecated: true.` being in the wrong order made it not appear as deprecated in the generated docs.